### DISCUSSION TITLE
#55 togETHer biweekly meeting

### DISCUSSION THREAD

**Discussion by 0xPabloLI** [tog-ETH-er/togETHer#40] [General]
Agenda:

-Community updates
-[Website update review](https://drive.google.com/file/d/1q2N9iHGUTB2YtoeXc2DfZOVB68uMWk8T/view)
-Fireside discussion

会议主题：togETHer
会议时间： 21:00-22:00
重复周期：每两周 (周一)

点击链接入会：
https://meet.google.com/uin-uwpc-gta

**Comment by 0xPabloLI** [comment]
PlanckerDAO

2023.1.7 [Dapp Learning跨社区分享：Account Abstraction and Guardian mechanism talk](https://plancker.notion.site/Account-Abstraction-and-Guardian-mechanism-talk-99d88cdc48c840708d7b9650e20faea5)
2022.12.31 [读 Vitalik: 
What in the Ethereum application ecosystem excites me（下）](https://plancker.notion.site/Vitalik-What-in-the-Ethereum-application-ecosystem-excites-me-c1985fb4d5404c82ac5ae3aeec962893)

**Comment by Thegaram** [comment]
### Crypto Tech Night

12.15 CTN#19: Account Abstraction and MPC Wallets (together with PlanckerDAO) https://www.bilibili.com/video/BV1o44y1d7G7

01.11 CTN Let'sTalk#4: 如何制作WEB3科普内容？https://twitter.com/CryptoTechNight/status/1612392508440608768

![image](https://user-images.githubusercontent.com/7571518/211301702-7930ce4b-701b-40c0-8e14-0daa38011deb.png)

**Comment by stephanieeee-ho** [comment]
### ECN
1. 以太坊2022年度盘点
    https://www.ethereum.cn/ecosystem/ethereum-in-2022
2. AllCoreDevs 更新 014
    https://www.ethereum.cn/Eth2/allcoredev-update014
3. 为什么 Paradigm 错了：论 rETH 将如何逆袭 stETH
    https://www.ethereum.cn/Staking/why-paradigm-was-wrong
4. 第五期 Podcast Ep.5 FTX 爆雷事件引发了对于资产安全的思考
    https://www.xiaoyuzhoufm.com/episode/63b596d10504f712092d5f9d?s=eyJ1IjoiNjJhZGY1MzNlZGNlNjcxMDRhODU1NWZkIiwiZCI6M30%3D

**Comment by 0x-stan** [comment]
# Dapp-Learning

## repo update

5个PR #909 ～ #914

https://github.com/Dapp-Learning-DAO/Dapp-Learning/pulls

## sharing meeting

近两周举办了3场分享会

- 《Issuing bonds using debond protocol》 by Yunan LIU
  - https://www.youtube.com/watch?v=MJ8th9Jyatc

- 《halo2 applied in zkEVM》by Dream
  - https://www.youtube.com/watch?v=yvNu7iAkgSo
  - https://www.youtube.com/watch?v=0fmi09xq2vA
  
- 《Account Abstraction and Guardian mechanism talk》by Survivor
  -  https://www.youtube.com/watch?v=qhSXyBW-7WE

**Comment by luckyyang** [comment]
### Rebase
- Web3 daily 更新
- 黑客松项目系列分享，7 场
- 1.6 成都 workshop

**Comment by Stamp9** [comment]
### 原语里弄

新研究专题「寻找密码朋克」 发布：https://www.primitiveslane.org/cryptographers
「寻找密码朋克」首篇文章「奥古斯特·柯克霍夫传」发布：https://www.primitiveslane.org/post/auguste_kerckhoffs

**Comment by Riely007** [comment]
### 翻译小组 

- 日常翻译进展中
